### PR TITLE
recommmended action during bull market

### PR DESCRIPTION
**Pull request by hjchan**

### CODE REVIEW

File: labs/2-perpetual-exchange/3-jlp-pool.md [hunk 1]
```diff
@@ -35,6 +36,9 @@ Now, let's explore some of the risks associated with holding JLP:
 
 **Profit and Loss (P&L) Dynamics:** Traders' P&L from perpetual trading impacts the JLP pool. If a trader incurs a net positive P&L, the losses are sourced from the JLP pool to compensate the trader. Conversely, if a trader's P&L is net negative, the gains are deposited into the JLP pool for LP holders.
 
+JLP pool will lose in token value but not in USD value because the underlying token value in the pool appreciate in value as well. Read more about a reasearch from community member on how this work.
+https://skribr.io/app/article/exploring-jupiters-perpetual-futures-a-comprehensive-research-analysis/
```
Comment (siong1987) [inline labs/2-perpetual-exchange/3-jlp-pool.md:40]: pull the graph to here and point to this article to ask the users to read more there.

File: labs/2-perpetual-exchange/3-jlp-pool.md [hunk 2]
```diff
@@ -126,3 +130,7 @@ revenue share you generate = $35,000 x 0.025% = $8.75
 ```
 
 Every Monday, at ~ UTC 18:00, the Estimated APY figure is updated with the above calculation, by using the previous week's fees and estimating an APR and APY for users.
+
+### Recommended action during bull market
+
+JLP is denominated in USD value. During bull market, JLP will unlikely to outperform SOL, ETH or BTC. It is recommended to swap stable token into JLP and earn the yield by trading activities.
```
Comment (siong1987) [inline labs/2-perpetual-exchange/3-jlp-pool.md:136]: just put this under Risk. no need a new section.

Comment (siong1987) [reply labs/2-perpetual-exchange/3-jlp-pool.md:136]:
```
JLP is denominated in USD. During bull market, JLP is unlikely to outperform SOL, ETH or BTC. It is recommended to swap stable token (USDC/USDT) into JLP and earn the yield from the perps trading activities.
```

Comment (julianhzhu) [reply labs/2-perpetual-exchange/3-jlp-pool.md]: Recommended for whom? This language reads like financial advice to me.

**Review by benlvb — APPROVED** [review]

**Pull request merged**